arcjet-nest/index.ts

```diff
@@ -395,7 +433,12 @@ let ArcjetGuard = class ArcjetGuard implements CanActivate {
       aj = rules.reduce((aj, rule) => aj.withRule(rule), aj);
     }
 
-    const request = context.switchToHttp().getRequest();
+    const request = requestFromContext(context);
+
+    // If we cannot access the request, we "fail open" by allowing the request
+    if (typeof request === "undefined") {
+      return true;
```

Comment:
Should we log a warning in this fail open case?

Reply:
We don't have access to the logger in the guard.

Reply:
https://github.com/arcjet/arcjet-js/issues/1859